scapy/contrib/openflow.py

```diff
@@ -549,7 +549,7 @@ class OFPTHello(_ofp_header):
                    ByteEnumField("type", 0, ofp_type),
                    ShortField("len", None),
                    IntField("xid", 0)]
-    overload_fields = {TCP: {"sport": 6653}}
+
```

Comment:
Is this related to this PR?

Reply:
To make it short, a few protocols used a bind on only one port (only sport, or only dport).
The other one is left to it's default value: 80. Therefore, the HTTP bind would be chosen rather than theirs.

To fix this, those protocols now bind the 2 ports